scripts/restart_ecs

```diff
@@ -12,7 +12,7 @@ healthcheck_timeout="180"
 
 cluster_name="${APP_ENV}-easi-app"
 service_name="easi-app"
-( set -x ; aws ecs update-service --cluster "$cluster_name" --service "$service_name" --force-new-deployment )
+( set -x ; aws ecs update-service --cluster "$cluster_name" --service "$service_name" --force-new-deployment ) | jq '.service.events = ["REDACTED"]'
```

Comment:
```suggestion
( set -x ; aws ecs update-service --cluster "$cluster_name" --service "$service_name" --force-new-deployment ) | jq '.service.events = ["OMITTED FOR BREVITY"]'
```
This is minor, but can you change the word "REDACTED" to something more neutral? "REDACTED" might lead some people in the future to think the events are sensitive information, which they're not.

Reply:
How about `REMOVED`?

Reply:
That works

Reply:
Could you also add a brief comment in the script explaining why we're removing the service events?